assets/src/blocks/Cookies/CookiesFrontend.js

```diff
@@ -153,6 +166,62 @@ export const CookiesFrontend = props => {
         />
       </Fragment>
       }
+      {((ENABLE_ANALYTICAL_COOKIES) && (isEditing || (analytical_cookies_name && analytical_cookies_description))) &&
+      <Fragment>
+        <label className="custom-control"
+              style={isSelected ? { pointerEvents: 'none' } : null}>
+          <input
+            type="checkbox"
+            tabIndex={isSelected ? '-1' : null}
+            name="analytical_cookies"
+            onChange={ () => {
+              if (analyticalCookiesChecked) {
+                setUserRevokedAnalytical(true);
+                setUserRevokedAllCookies(true);
+                if (allCookiesChecked) {
+                  setConsentCookie(ALL_COOKIES);
+                } else {
+                  setConsentCookie(ONLY_NECESSARY);
+                }
+              } else {
+                if (allCookiesChecked) {
+                  setConsentCookie(NECESSARY_ANALYTICAL_MARKETING);
+                } else {
+                  setConsentCookie(NECESSARY_ANALYTICAL);
+                }
+              }
+            } }
+            checked={analyticalCookiesChecked}
+            className="p4-custom-control-input"
+          />
+          <FrontendRichText
```

Comment:
This `FrontendRichText` component is not ideal, as it results in sending editor code that is not needed to visitors. It also complicates the component. But could be tricky to rewrite in this case. If you feel up for it, would be nice to address it. But otherwise we can also do this at a later point.